.github/workflows/ci.yml

```diff
@@ -49,22 +30,14 @@ jobs:
         run: |
           npm install -g npm@8.3.1
           npm install -g npm@latest
-      - name: Install bolt
-        shell: bash
-        run: |
-          case "$(uname -s)" in
-            Windows*|CYGWIN*|MINGW*|MSYS*) BOLT_VERSION=0.21.2 ;;
-            *) BOLT_VERSION=latest ;;
-          esac
-          npm install -g bolt@$BOLT_VERSION
       - name: Install
-        run: bolt
+        run: yarn && lerna bootstrap
```

Comment:
I'm 80% sure that we can just call `yarn install` here, since workspaces takes care of simlinking and hoisting, thus `lerna bootstrap` is equivalent to `yarn install`. 

would like some feedback here